src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -148,15 +158,15 @@ export default class ChoroplethLayer extends KibanaMapLayer {
         let errorMessage;
         if (e.status === 404) {
```

Comment:
Custom region layers defined in `kibana.yml` are failing to load. I set a breakpoint on this line and it looks like the `layerConfig` is empty by the time it gets to this point.

Reply:
ok, correct, this is a problem, I'll take a look